eng/CredScanSuppression.json

```diff
@@ -11,6 +11,7 @@
                 "kt#_gahr!@aGERDXA",
                 "Aa1!zyx_",
                 "Aa!1()-xyz",
+                ":code:`<pfx-file-password>`",
```

Comment:
Do you know if it needs this entire string or just `<pfx-file-password>`?

Reply:
Based on the row and column of the warning, it looks like it should be the whole string -- but verifying with a test run is the smart thing to do, thanks for starting that!